stakelab/chains.json

```diff
@@ -472,9 +463,9 @@
     },
     {
       "name": "shentu",
-      "address": "certikvaloper1795d6km32m8qtquepatf7npean9dgvrpr3le7q",
+      "address": "shentuvaloper1795d6km32m8qtquepatf7npean9dgvrpxfnz02",
       "restake": {
-        "address": "certik1mjq48r6435aewerpruwc8up3tz3rzan2t3zqq7",
+        "address": "",
```

Comment:
Blank address here - can you populate or remove the whole restake section if it's not needed?

Reply:
Done